stations.csv

```diff
@@ -4891,7 +4891,7 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;is_city;country;
 6489;Londoño;londono;7131406;;;;;f;ES;f;Europe/Madrid;f;ESADJ;;f;;f;;f;;f;;f;;f;;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
 6490;Adanero (Muñoz);adanero-munoz;7110407;;;;;f;ES;f;Europe/Madrid;f;ESADK;;f;;f;;f;;f;;f;;f;;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
 6491;Medellín;medellin;7137408;;;;;f;ES;f;Europe/Madrid;f;ESADL;;f;;f;;f;;f;;f;;f;;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
-6492;Aranda de Duero – Montecillo;aranda-de-duero-montecillo;7117200;;41.667778;-3.695836;;f;ES;f;Europe/Madrid;t;ESADM;;f;;f;;f;;f;;f;;f;;;f;;f;17200;t;;f;f;;;;;;;;;;;;;;;;;;
```

Comment:
I say yes, use regular dash. I cannot find any instances of the name written in that manner.